packages/plugin/src/rules/require-description.ts

```diff
@@ -95,6 +96,16 @@ const rule: GraphQLESLintRule<[RequireDescriptionRuleConfig]> = {
             }
           `,
         },
+        {
+          title: 'Correct',
+          usage: [{ operationFieldDefinition: true }],
+          code: /* GraphQL */ `
+            type Mutation {
+              "Create a new user"
+              createUser: User
+            }
```

Comment:
it will be better to include another type to show that description is only required for the root type field